package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "start": "npm run watch",
-    "postinstall": "npm run bootstrap",
+    "postinstall": "npm run bootstrap && opencollective-postinstall",
```

Comment:
Please remove `opencollective-postinstall` from the monorepo package.json.

Reply:
removed